contracts/W3BC6-Bank.sol

```diff
@@ -16,13 +16,20 @@ contract Bank {
         - apply DRY(Don't Repeat Yourself) to withdraw and transfer_to_address functions
         - check against 0x0000000000000000000000000000000000000000 address for transactions
     */
-     address immutable private owner;
-       constructor () {
-         owner = msg.sender;
-     }
+
+    address immutable private owner;
 
     mapping(address => uint) user_balance;
     uint private bank_balance; 
+    
+    constructor () {
+       owner = msg.sender;
+     }
+
+    function Deposit() payable public {
+        require(msg.value > 0, "Amount to be deposited is too small");
+        balance[msg.sender] += msg.value;
```

Comment:
Kindly take note of the bank_balance state variable, update accordingly on deposit.